nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -355,6 +355,7 @@ in
         ExecStart = "${wrappedSlurm}/bin/slurmd";
         PIDFile = "/run/slurmd.pid";
         ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
+        LimitMEMLOCK = "infinity";
```

Comment:
Why is this necessary?

Reply:
This fixed the issue of slurm not being able to use mpirun, while it worked in a shell. I'll try again to see if i've messed up my testing.